qemu/tests/s390x_cpu_model_boot.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot guest with supported cpu-models
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    boot_cpu_models = params['boot_cpu_models'].split(';')
+    params['start_vm'] = 'yes'
+    vm_name = params['main_vm']
+    vm = env.get_vm(vm_name)
+    args = {'type': 'static', 'model': {'name': 'host'}}
+    output = vm.monitor.cmd('query-cpu-model-expansion', args)
+    host_model = output.get('model')
+    host_model_name = host_model.get('name')[:-5]
+    for boot_models in boot_cpu_models:
+        if host_model_name in boot_models:
+            boot_cpu_models = boot_models.split(',')
+    for boot_cpu_model in boot_cpu_models:
+        params["cpu_model"] = boot_cpu_model
+        try:
+            test.log.info("Start boot guest with cpu model: %s." %
+                          boot_cpu_model)
+            vm.create(params=params)
```

Comment:
So the create here is another vm, right? 
If it is, we should use another name. for example: vm2

Reply:
solved